src/app/components/Pagination/index.tsx

```diff
@@ -0,0 +1,203 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useContext } from 'react';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '#app/legacy/psammead/gel-foundations/src/breakpoints';
+import { RequestContext } from '#app/contexts/RequestContext';
+import { LIVE_PAGE } from '#app/routes/utils/pageTypes';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import buildBlocks, { VISIBILITY } from './buildBlocks';
+import { Ellipsis, LeftChevron, RightChevron } from '../icons';
+import VisuallyHiddenText from '../VisuallyHiddenText';
+import styles from './index.styles';
+
+interface LinkComponentProps {
+  children: number | JSX.Element;
+  pageNumber: number;
+  isActive?: boolean;
+}
+
+interface ArrowProps {
+  activePage: number;
+  children: string | JSX.Element;
+  dir: string;
```

Comment:
We have a `Direction` type that could be used here: https://github.com/bbc/simorgh/blob/729de2d6ab0b0244d0e567433430a2bf476b70a0/src/app/models/types/global.ts#L6-L7